docs/config-presets-and-options.md

```diff
@@ -0,0 +1,56 @@
+# Configuration Presets and Options
+
+## Defining the Cluster Topology
+
+Openshift clusters can have different topologies. In Openshift development we refer to 
+these different topologies as different installations with adjectives and acronyms. 
+Below is the list of cluster toplogies currently being used to test the installers:
+
+|                       | Master Nodes | Worker Nodes | Masters <br>Schedulable |
+|-----------------------|--------------|--------------|-------------------------|
+| High Availablity (HA) |       3      |       2      |           No            |
```

Comment:
note: we could have potentially even more than 2 nodes, that's the min support amount (ie `2+`)